src/acl/external/kerberos_sid_group/ext_kerberos_sid_group_acl.pl.in

```diff
@@ -59,7 +59,7 @@ AD group name to be used for SID lookup. List separated by a colon (:)
 
 =head1 CONFIGURATION
 
-  auth_param negotiate program /path/to/negotiate_wrapper_auth -d 
+  auth_param negotiate program /path/to/negotiate_wrapper_auth -d
```

Comment:
these are missing the '\' at end of lines to indicate wrapping.

Reply:
hmm, github does not seem to like showing slashes in comments.

Reply:
Sorry I don't follow. Is there something I should change ?

Reply:
I believe @yadij is implying that you should add backslashes (`\`) at end of this and probably other similar lines to indicate command/line wrapping.